lib/ui/widgets/schedule_settings.dart

```diff
@@ -0,0 +1,172 @@
+                    child: Container(
+                      height: 5,
+                      width: 50,
+                      decoration: const BoxDecoration(
+                          color: Colors.grey,
+                          borderRadius: BorderRadius.all(Radius.circular(8.0))),
+                    ),
+                  ),
+                ),
+              Padding(
+                  padding: const EdgeInsets.fromLTRB(15, 20, 20, 20),
+                  child: Text(AppIntl.of(context).schedule_settings_title,
+                      style: Theme.of(context).textTheme.headline6)),
+              Expanded(
+                child: ListTileTheme(
+                  selectedColor: Theme.of(context).textTheme.bodyText1.color,
+                  child: ListView(
+                      children: _buildSettings(
+                          context, model as ScheduleSettingsViewModel)),
+                ),
+              ),
+            ],
+          ),
+        ),
+      );
+
+  List<Widget> _buildSettings(
+      BuildContext context, ScheduleSettingsViewModel model) {
+    final list = _buildCalendarFormatSection(context, model);
+
+    // TODO Uncomment when https://github.com/aleksanderwozniak/table_calendar/issues/164 is close
+    //list.addAll(_buildStartingDaySection(context, model));
+
+    list.addAll(_buildShowTodayButtonSection(context, model));
+
+    return list;
+  }
+
+  List<Widget> _buildShowTodayButtonSection(
+          BuildContext context, ScheduleSettingsViewModel model) =>
+      [
+        ListTile(
+          trailing: Switch(
+            value: model.showTodayBtn,
+            onChanged: (value) => model.showTodayBtn = value,
+            activeColor: AppTheme.etsLightRed,
+          ),
+          title:
+              Text(AppIntl.of(context).schedule_settings_show_today_btn_pref),
+        ),
+        const Divider(thickness: 1)
+      ];
+
+  List<Widget> _buildCalendarFormatSection(
+      BuildContext context, ScheduleSettingsViewModel model) {
+    final tiles = [
+      Padding(
+        padding: const EdgeInsets.only(left: 15.0, right: 15.0, bottom: 2.0),
+        child: Text(
+          AppIntl.of(context).schedule_settings_calendar_format_pref,
+          style: TextStyle(
+            color: Theme.of(context).accentColor,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ),
+      const Divider(endIndent: 50, thickness: 1.5)
+    ];
+
+    for (final CalendarFormat format in model.calendarFormatPossibles) {
+      tiles.add(ListTile(
+        selected: model.calendarFormat == format,
+        selectedTileColor: selectedColor,
+        onTap: () => setState(() => model.calendarFormat = format),
+        title: Text(getTextForFormat(context, format)),
+      ));
+    }
+
+    tiles.add(const Divider(thickness: 1));
+
+    return tiles;
+  }
+
+  // TODO remove ignore when https://github.com/aleksanderwozniak/table_calendar/issues/164 is close
```

Comment:
reminder @apomalyn

Reply:
Same as the precedent.  This TODO should stay there until the PR in tablecalendar is closed but nice catch! Nothing pass your eyes ahha :D